board/m2k/S21misc

```diff
@@ -11,6 +11,10 @@ handle_calibration_files() {
 	elif [[ -s /mnt_jffs2/${CALIBFILENAME_FACTORY} ]]; then
 		cp /mnt_jffs2/${CALIBFILENAME_FACTORY} /opt/${CALIBFILENAME}
 	fi
+
+	if [[ -s /mnt_jffs2/${CALIBFILENAME_TEMP_LUT} ]]; then
```

Comment:
i'm puzzled; isn't this `/mnt/jffs2` now?
or i am confusing something?

Reply:
i think for pluto this may be `/mnt/jffs2`

Reply:
It should be /mnt/jffs2 now for both - looks like this file wasn't converted yet.

Reply:
Thank you for catching this. This was an older branch and forgot to rebase it on master.